client/src/Components/Signup/style.css

```diff
@@ -0,0 +1,117 @@
+.content-signup{
+    margin: 0 auto;
+    width: 358px;
+    height: auto;
+    box-sizing: border-box;
+    background: #FFFFFF;
+    box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.5);
+    border-radius: 2px;
+    padding: 15px;
+    margin: 50px auto;
+    }
+    .content-signup__word-sigup{
+    font-family: Lato;
```

Comment:
why you put  `    font-family: Lato; ` 
when did we add it on the public style?